config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  post 'ups' => 'services#ship', service: 'ups'
+  post 'usps' => 'services#ship', service: 'usps'
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
```

Comment:
It's a good idea to clear out unnecessary generated comments, to not distract from your code or your in-code documentation comments.